src/components/slider/slider.tsx

```diff
@@ -175,7 +175,7 @@ export class Slider {
                         aria-valuenow={this.multiplyByFactor(this.getValue())}
                         aria-label={this.label}
                         aria-disabled={this.disabled || this.readonly}
-                        data-step={this.step}
+                        data-step={this.multiplyByFactor(this.step)}
```

Comment:
Yeah, it probably makes more sense like this… 😊👍

This is a breaking change though, so we should mark it as such, to get a new major version when this is merged.

Reply:
(I can fix it.)